OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -187,6 +190,8 @@ public AutoTarget(ActorInitializer init, AutoTargetInfo info)
 				stance = self.Owner.IsBot || !self.Owner.Playable ? info.InitialStanceAI : info.InitialStance;
 
 			PredictedStance = stance;
+
+			allowMovement = Info.AllowMovement && self.TraitOrDefault<IMove>() != null;
```

Comment:
@abcdefg30 can you elaborate on this change? If we're not checking for disabled traits, wouldn't it be easier to just set `AllowMovement: False` on the handful of defensive structures?

Reply:
The idea was that it does not make sense to even consider moving when there is no `IMove` trait that can be used to move. We can of course still set `AllowMovement: False` explicitly on structures.